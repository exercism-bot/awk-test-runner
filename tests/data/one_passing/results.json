{
  "version": 2,
  "status": "pass",
  "test-environment": {
    "gawk": "GNU Awk 5.1.0, API: 3.0 (GNU MPFR 4.1.0, GNU MP 6.2.1)",
    "bats": "Bats 1.7.0",
    "OS": "Ubuntu 22.04 LTS"
  },
  "tests": [
    {
      "name": "say hello",
      "status": "pass",
      "test_code": "run bash one_passing.sh\n[ \"$status\" -eq 0 ]\n[ \"$output\" == \"Hello, World!\" ]"
    }
  ]
}
