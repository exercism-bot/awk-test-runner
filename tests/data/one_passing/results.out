1..1
ok 1 say hello
